index.js

```diff
@@ -68,7 +68,7 @@ module.exports = function(version, _options) {
 
             return config.join('');
         },
-        compile: function(language, step) {
+        compile: function(language, step, legOptions) {
```

Comment:
should we keep this with `options` only? That would be consistent with the `options` we already have for the module initialization.